item.rb

```diff
@@ -1,36 +1,60 @@
 require 'date'
+require './label_methods'
+require 'securerandom'
 
 class Item
-  attr_reader :id
-  attr_accessor :genre, :label, :published_date :archived
+  attr_reader :id, :author
+  attr_accessor :genre, :source, :label, :publish_date, :archived
 
-  def initialize(id:, published_date: archived: false)
-    @id = :id
+  def initialize(id:, publish_date:, archived: false)
+    @id = id || SecureRandom.hex(4)
     @genre = []
     @author = []
     @label = []
-    @published_date = Date.parse(published_date)
+    @publish_date = Date.parse(publish_date)
     @archived = archived
   end
 
   def add_genre(genre)
-    @genre.push(genre) unless @genre.include?(genre)
+    @genre = genre
+    genre&.items&.push(self)
   end
 
-  def add_author(author)
```

Comment:
Once again, please try to pull the existing changes from the dev repo before pushing. This is deleting the author method(from #36 ) already working on the current dev implementation.

Let me know if you need help conciling the conflicts so we have most of our two features working after this PR.